schemas/mapping.schema.json

```diff
@@ -0,0 +1,37 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "id": "https://immersive-web/gamepad-mapping/0.1.0/mapping.schema.json",
+    "type": "object",
+    "description": "The root object for a gamepad mapping.",
+    "additionalProperties": false,
+    "required": [ "version", "gamepad" ],
+    "properties": {
+        "version" : {
+            "description": "The schema version",
+            "type" : "string"
+        },
+        "gamepad" : {
```

Comment:
That's actually what I had in the original design.  I changed it to this approach because I was simplifying the look-up process.  Developers know what folder to look for based on the gamepad id, which also makes it backwards compatible for WebVR :D

Reply:
I've filed #5 to address this